CONTRIBUTING.md

```diff
@@ -0,0 +1,93 @@
+Contributing to Flutter
```

Comment:
Rather than duplicating this, I suggest we link to it, similar to what we do over in flutter/plugins: https://github.com/flutter/plugins#contributing

Reply:
Actually the link you gave is to [README.md](https://github.com/flutter/plugins/blob/master/README.md) in the plugins dir.  The [CONTRIBUTING.md](https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md) there is basically the same as here (I did start from that one to make this one). This one is customized for the packages repo.

I'll update the README.md to be like the one in plugins.